app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::API
   include ActionController::HttpAuthentication::Token::ControllerMethods
 
   attr_reader :current_user
+  helper_method :current_user
```

Comment:
I see that you did this to enable `current_user` available in the view, right? We might have no need for the `attr_reader` then :thought_balloon: